drivers/entropy/Kconfig.litex

```diff
@@ -0,0 +1,14 @@
+# Kconfig.litex - PRBS entropy generator driver configuration
+#
+# Copyright (c) 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
New formatting for the Kconfig header comments

https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#header-comments-and-other-nits

Reply:
Formatting changed.